psr/README-PSR.md

```diff
@@ -0,0 +1,46 @@
+
+
+A Proposal for Proposing Specification Changes
```

Comment:
```suggestion
A Proposal for  Specification Changes
```

Reply:
I was being cheeky.  :)  But yes, I will remove it.